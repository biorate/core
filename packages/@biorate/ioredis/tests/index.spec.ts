import { expect } from 'chai';
import { root } from './__mocks__';

describe('@biorate/ioredis', function () {
  this.timeout(3e4);

  before(async () => await root.$run());

  it('set', async () => {
    expect(await root.connector.current!.set('key', 'value')).toMatchSnapshot();
  });

  it('get', async () => {
    expect(await root.connector.current!.get('key')).toMatchSnapshot();
  });

  it('delete', async () => {
    expect(await root.connector.current!.del('key')).toMatchSnapshot();
  });
});
