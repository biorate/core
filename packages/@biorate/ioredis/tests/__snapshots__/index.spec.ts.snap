// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@biorate/ioredis delete 1`] = `1`;

exports[`@biorate/ioredis get 1`] = `"value"`;

exports[`@biorate/ioredis set 1`] = `"OK"`;
